.github/scripts/leaderboard.js

```diff
@@ -0,0 +1,186 @@
+const axios = require('axios');
+const fs = require('fs').promises;
+const path = require('path');
+const { parse } = require('csv-parse');
+const crypto = require('crypto');
+
+const CONFIG = {
```

Comment:
I love how you made this config, with fallbacks, dev-purposed JSON files and necessary settings, clean code Justin!